index.html

```diff
@@ -237,6 +237,22 @@ <h1 class="description-title">What is Real Dev Squad?</h1>
         </div>
       </section>
 
+      <section class="section-features hide" id="section-join">
+        <h1 class="features-title">How to Join Real Dev Squad</h1>
+        <div class="join-description">
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Dolorem
+            doloribus repudiandae aliquam blanditiis molestias impedit
+            voluptates qui deleniti laboriosam nesciunt? Quas ullam consequuntur
+            assumenda molestiae dolorem ut iusto pariatur corporis porro harum
+            accusamus sint veniam et dignissimos asperiores, id velit expedita
+            officiis labore libero possimus. Incidunt suscipit esse vero
+            provident.
```

Comment:
why is this here?

Reply:
Since the content wasn't provided to me.